src/client/src/components/pages/managers/addManager/index.jsx

```diff
@@ -24,17 +26,57 @@ export default class index extends Component {
     this.setState(fields);
   };
 
+  addManager = async obj => {
+    const confirm = await swal({
+      type: "warning",
+      html: "Are you sure that you want to add this manager ?",
+      showCancelButton: true,
+      focusConfirm: false,
+      confirmButtonText: '<i className="fa fa-thumbs-up"></i> Yes',
```

Comment:
`confirmButtonText` : its a text.
so I think that you might replace `className` with `class` to show the `fa-thump-up` or `fa-thump-down` icons in your view.